src/Mod/Path/Gui/Resources/Path.qrc

```diff
@@ -66,6 +66,7 @@
         <file>icons/Path-ToolController.svg</file>
         <file>icons/Path-Toolpath.svg</file>
         <file>icons/Path-ToolTable.svg</file>
+	<file>icons/Path-Vcarve.svg</file>
```

Comment:
There's a tab here, it should be spaces.

Reply:
fixed